Sources/protoc-gen-grpc-swift/Generator.swift

```diff
@@ -99,9 +99,6 @@ class Generator {
     // mappings to import the service protos as well as and any other proto modules that the file
     // imports.
     let moduleMappings = options.protoToModuleMappings
-    if let serviceProtoModuleName = moduleMappings.moduleName(forFile: file) {
```

Comment:
cc @allevato who added this originally (https://github.com/grpc/grpc-swift/pull/347)

My understanding of the comment above is that if we have a foo.proto containing:

```proto
service FooService { ... }

message FooMessage { ... }
```

Then Bazel would produce two Swift modules, say `FooServices` and `FooMessages`; and this line is to import the `FooMessages` module. 

If my understanding of this is correct this seems like a misuse of module mappings since the output generated from a single proto is split across multiple modules. Using it in this way also leads to an unnecessary warning if you want to use module mappings in the intended way. Can this be better expressed in the Bazel rules as an additional import instead of module mappings?

Reply:
You're correct about the original motivation; the Bazel rule `swift_proto_library` generates just the code for the messages, the rule `swift_grpc_library` generates just the code for the services, and they're in separate modules so the latter needs to import the former.

The `ExtraModuleImports` flag was added after my change in https://github.com/grpc/grpc-swift/pull/402 to support getting the right imports for client stubs, so that wasn't an option when we first spun up Bazel support. But looking at the rule implementation again (it's been a while), I think it should be a straightforward change to use `ExtraModuleImports` in the same way here, so I don't foresee a problem with this change; we'll just need to update the rules when we update to this version.

cc @mpetrov for awareness as well.

Reply:
Awesome, thanks very much.

As a heads up: this is change is for `protoc-gen-grpc-swift` (gRPC Swift 1.x);  `protoc-gen-swiftgrpc` (gRPC Swift 0.x) won't be affected.